source/installation/with/yunohost.html.md

```diff
@@ -2,8 +2,19 @@
 title: Installing with YunoHost
 ---
 
-This installation method is not yet documented. Your help would be welcome!
+You must have a Yunohost server.
```

Comment:
"you need" might sound a bit better here.?

Reply:
Yes without problem, made the change, my english is not subtle enough ^^